x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/WaterfallContainer/WaterfallContainer.stories.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { TraceAPIResponse } from '../../../../../../../../../plugins/apm/server/lib/traces/get_trace';
+import { WaterfallContainer } from './index';
+import {
+  location,
+  urlParams,
+  simpleTrace,
+  traceWithErrors,
+  traceChildStartBeforeParent
+} from './waterfallContainer.stories.data';
+import { getWaterfall } from './Waterfall/waterfall_helpers/waterfall_helpers';
+
+storiesOf('app/Transaction/Waterfall', module).add('simple', () => {
+  const waterfall = getWaterfall(
+    simpleTrace as TraceAPIResponse,
+    '975c8d5bfd1dd20b'
+  );
+  return (
+    <WaterfallContainer
+      location={location}
+      urlParams={urlParams}
+      waterfall={waterfall}
+      exceedsMax={false}
+    />
+  );
+});
+
+storiesOf('app/Transaction/Waterfall', module).add('with errors', () => {
+  const waterfall = getWaterfall(
+    (traceWithErrors as unknown) as TraceAPIResponse,
+    '975c8d5bfd1dd20b'
+  );
+  return (
+    <WaterfallContainer
+      location={location}
+      urlParams={urlParams}
+      waterfall={waterfall}
+      exceedsMax={false}
+    />
+  );
+});
```

Comment:
I don't think it's useful to show the whole source here. What do you think?

```suggestion
}, { info: { source: false } });
```

Reply:
thanks @smith I didn't know about that property.